x-pack/plugins/cases/common/api/cases/comment/files.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as rt from 'io-ts';
+
+export const FileAttachmentMetadataRt = rt.type({
```

Comment:
To be future-proof let's change it to an array of `file` objects. Even if we use only one, we may want to support multiple ones in the future.